backend/api/chatLink/utils/link.js

```diff
@@ -1,18 +1,19 @@
 const uuidv4 = require('uuid').v4;
-const { generatePIN } = require('./pin')
+const { generatePIN } = require('./pin');
 
 const chatLinkDomain = process.env.CHAT_LINK_DOMAIN;
 
 const generateLink = () => {
   const hash = uuidv4();
+  const pinLength = 4;
```

Comment:
As it's just a constant let's move it top, out of function and name it something like `PIN_LENGTH`